client/apps/shipping-label/view-wrapper-label.js

```diff
@@ -125,11 +125,21 @@ class ShippingLabelViewWrapper extends Component {
 
 			// All items are packaged, just show track button
 			return (
-				<Button
-					onClick={ this.handleTrackPackagesButtonClick }
-				>
-					{ translate( 'Track Packages' ) }
-				</Button>
+				<span> 
+					<span className="shipping-label__redo-shipping-button">
+						<Button
+							borderless
+							onClick={ this.handleCreateLabelButtonClick }
+						>
+							{ translate( 'Redo Shipping' ) }
```

Comment:
We need to change this label per @aleftick suggestion

```suggestion
							{ translate( 'Create new label' ) }
```